webservices/resources/sched_c.py

```diff
@@ -0,0 +1,76 @@
+import sqlalchemy as sa
+from flask_apispec import doc
+
+from webservices import args
+from webservices import docs
+from webservices import utils
+from webservices import filters
+from webservices import schemas
+from webservices.common import models
+from webservices.common.views import ItemizedResource
+
+
+@doc(
+    tags=['loans'],
+    description=docs.SCHEDULE_C,
+)
+class ScheduleCView(ItemizedResource):
+
+    model = models.ScheduleC
+    schema = schemas.ScheduleCSchema
+    page_schema = schemas.ScheduleCPageSchema
+    """
+    @property
+    def year_column(self):
+        return self.model.two_year_transaction_period
+    """
+    @property
+    def index_column(self):
+        return self.model.sub_id
+
+    filter_multi_fields = [
+        ('image_number', models.ScheduleC.image_number),
+        ('committee_id', models.ScheduleC.committee_id),
+    ]
+
+    filter_fulltext_fields = [
+        ('loaner_name', models.ScheduleC.loan_source_name),
```

Comment:
@LindsayYoung adding a `TSVECTOR` for this columns won't be a simple thing to do, not how it's currently implemented.  For the other itemized tables we map to the `ofec...` tables, and in those tables we create a `TSVECTOR` column for fast full-text searching.

So to add that support for this we'd have to add a new TSVECTOR column and keep it update with a trigger, as documented in this reference: https://blog.lateral.io/2015/05/full-text-search-in-milliseconds-with-postgresql/

Not a hard thing to do at all, but won't be able to get in this release, so I'll just pull that filter for now and we can get it in for the release possible.